### PR TITLE
The most basic admin styling

### PR DESCRIPTION
**Pull request by adam-hart**
This adds the most basic of admin page styling.  It will also make open-referral usable with the change to Location.js (bug somewhere in @haabaato's code).

@zendesk/volunteer

### CODE REVIEW

File: components/Location/Location.js [hunk 1]
```diff
@@ -217,7 +217,7 @@ const Location = (props) => {
             <p className={s.serviceDescription}>{service.description}</p>
             <table className={s.openHours}>
               <tbody>
-                {getSchedule(service.schedules)}
+              {/* {getSchedule(service.schedules)} */}
```
Comment (roseleaf) [inline components/Location/Location.js]: @haabaato this is the bug

Comment (thebritican) [inline components/Location/Location.js]: 🐛

**Comment by adam-hart** [comment]
BOOSH

File: components/LocationEdit/LocationEdit.css [hunk 2]
```diff
@@ -48,12 +54,43 @@
   align-items: center;
 }
 
+.addToSubsection {
+  color: #666;
+  padding: 0;
+  margin: 0 10px;
```
Comment (thebritican) [inline components/LocationEdit/LocationEdit.css]: hey yo maybe no margin :P

File: components/LocationEdit/LocationEdit.js [hunk 3]
```diff
@@ -131,23 +136,37 @@ class LocationEdit extends Component {
...
+            ))}
+            <button
+              className={s.addToSubsection}
+              onClick={this.newService}
+              title={`Click to add a new service`}>
+              + Add
+            </button>
+          </div>
+          <div className={s.serviceEdit}>
+            {selectedService ?
```
Comment (thebritican) [inline components/LocationEdit/LocationEdit.js]: can be
```javascript
selectedService && <ServiceEdit ... />
```

File: components/Location/Location.js (hunk 1 above)
Comment (haabaato) [inline components/Location/Location.js]: arrr, what's wrong with it?

**Pull request merged**